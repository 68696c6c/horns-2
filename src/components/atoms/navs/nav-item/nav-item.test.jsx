/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import NavItem from '.'

describe('NavItem', () => {
  it('should render as default', () => {
    const { container } = render(<NavItem>example</NavItem>)
    expect(container).toMatchSnapshot()
  })
})
