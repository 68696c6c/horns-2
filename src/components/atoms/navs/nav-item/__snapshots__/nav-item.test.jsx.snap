// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem should render as default 1`] = `
<div>
  <a
    class="nav-item css-1e72kte"
    color="background"
    cursor="pointer"
    href="#"
  >
    example
  </a>
</div>
`;
